src/component/aside/Aside.jsx

```diff
@@ -0,0 +1,55 @@
+const Aside = () => {
+  return (
+    <>
+      <aside className="side-section" id="side-section">
+        <div className="navbar-links">
+          <div className="sidebar-head">
+            <i className="fa fa-times" id="sidebar-close"></i>
+          </div>
+          <div className="avatar centered mb-lg">
+            <img
+              className="img-md border-round img-border"
+              src="./assets/toy1.jpg"
+              alt="Avatar"
+            />
+          </div>
+          <hr />
+          <ul className="navbar-items">
+            <li>
+              <a className="active-item decor-none" href="home.html">
```

Comment:
I think this is coming from html that's why anchor tag is here right?

Reply:
Yeah, will change that later